backend/layers/thirdparty/crossref_provider.py

```diff
@@ -159,3 +160,23 @@ def fetch_preprint_published_doi(self, doi):
                     return published_doi[0]["id"]
             except Exception:
                 pass
+
+
+class MockCrossrefProvider(CrossrefProviderInterface):
```

Comment:
Needed to fix tests that generate a collections as a result of making the interfaces protocol